infrahub_sdk/query_groups.py

```diff
@@ -94,7 +94,7 @@ async def get_group(self, store_peers: bool = False) -> InfrahubNode | None:
         if not store_peers:
             return group
 
-        self.previous_members = group.members.peers  # type: ignore[attr-defined]
+        self.previous_members = group._relationship_cardinality_many_data["members"].peers
```

Comment:
this one feels wrong to me, I think we should either use the Protocol or we should add a public method to access an attribute or a relationship by its name

```python 
group.get_relationship_many(name="members").peers
```